### ISSUE TITLE
lang and xml:lang match (5b7ae0): accessibility support and requirements disagree

### ISSUE THREAD

**Issue by Jym77** [act-rules/act-rules.github.io#659]
Rule https://act-rules.github.io/rules/5b7ae0 checks that `lang` and `xml-lang` match.

In the Accessibility Requirements, we have the fail-on-fail relation with SC 3.1.1: if the rule fails, so does 3.1.1.

But in the Accessibility Support, we say that "violation of this rule may not necessarily be a violation of [SC 3.1.1]".

The sentence in the Accessibility Support seems to be in direct contradiction with what is written in the Accessibility Requirements: does failing this rule fails SC 3.1.1 (as the AR say) or maybe not (as the AS say)?

**Comment by EmmaJP** [comment]
Refer comments in #668

**Comment by WilcoFiers** [comment]
@Jym77 This is not an issue. The intent of an assumption is to indicate when a rule might produce false positives. You can think of them as preconditions. If all assumptions are true, the failing the rule means the accessibility requirement is not satisfied.

**Comment by Jym77** [comment]
@WilcoFiers the discrepancy is not about the assumption but between the Accessibility Support and the Requirement.

* Requirement/outcome mapping: failing this rule results in SC 3.1.1 being not satisfied (usual fail-on-fail relation).
* Accessibility support: "violation [which, I guess, means failing] of this rule may not necessarily be a violation [which, I guess, means not satisfying] of WCAG 2"

The (only) assumption in that rule is about secondary language subtag. The Accessibility Support is about AT consistently using either only `lang` or only `xml:lang`.

So, the assumption can be true (inappropriate secondary subtag doesn't matter), and the rule fails (`lang` and `xml:lang` mismatch), but the Accessibility Support still says that I'm maybe still OK with WCAG (satisfying SC 3.1.1).

Maybe the solution is to add an Assumption in the line of "this rule assumes that both `lang` and `xml:lang` are used to decide language in a non-deterministic way by the AT"

**Comment by WilcoFiers** [comment]
@Jym77 This gets into a nuance of WCAG testing. See: [WCAG 2.1, section 5.2.4 Only Accessibility-Supported Ways of Using Technologies](https://www.w3.org/TR/WCAG21/#cc4) I can probably write a whole essay about why this is correct, but the short version is this.

The tester decides which assistive technologies they are interested in. This is a baseline they set. If something doesn't work in that baseline, and that "something" is relied upon for a success criterion to pass, then it's not accessible, and the SC fails. If the accessibility support baseline changes, the outcome of a test can change as well. That's not an assumption made by the rule, it's something that's baked into WCAG itself.

**Comment by Jym77** [comment]
I think I see your point (OK, I asked Kasper for more details…)
I still find the formulation a bit weird, also the fact that we do not have similar warning for any other rule seems unlikely… But I have nothing better to suggest, so I guess it's OK.
